feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.ts

```diff
@@ -297,4 +311,122 @@ export class ConfiguratorStorefrontUtilsService {
       ) as HTMLElement;
     }
   }
+
+  /**
+   * Retrieves a list of HTML elements based on querySelector when running in browser
+   *
+   * @param {string} querySelector - querySelector
+   * @returns {HTMLElement[] | undefined} - List of HTML elements
+   */
+  getElements(querySelector: string): HTMLElement[] | undefined {
+    if (this.windowRef.isBrowser()) {
```

Comment:
do we need this safeguard at all?
What is the use case, where this gets executed and we are NOT in the browser?

Reply:
As far as I understand by means of this condition `this.windowRef.isBrowser()`, we want to guarantee that this method is called only for the browser

Reply:
If we remove this condition here, then the customer will ask him/herself why in other method we still have this condition.

Reply:
I strongly believe that this condition does not have any impact on the performance.